oci/private/pull.bzl

```diff
@@ -490,9 +490,7 @@ buildozer 'set digest "sha256:{digest}"' 'remove tag' 'remove platforms' {option
             name = rctx.attr.name,
             target_name = rctx.attr.target_name,
             platform_map = {
-                # Workaround bug in Bazel 6.1.0, see
-                # https://github.com/bazel-contrib/rules_oci/issues/221
-                str(k).replace("@@", "@"): v
```

Comment:
The fix seems correct, even if it doesn't work here :-D. Is this running with a `WORKSPACE.bzlmod`? It could be picking up `platforms` from `WORKSPACE`.

Reply:
IIUC this should always have been `str(k): v`. I don't understand why that causes #211 to fail... more digging is likely needed.

Reply:
Bazel 6.2.0 contains the fix, I think it's fine to say rules_oci users need to upgrade. Thanks!